setup/flavors/compose/mailu.env

```diff
@@ -29,7 +29,7 @@ POSTMASTER={{ postmaster }}
 # Choose how secure connections will behave (value: letsencrypt, cert, notls, mail, mail-letsencrypt)
 TLS_FLAVOR={{ tls_flavor }}
 
-# Authentication rate limit per IP (per /24 on ipv4 and /48 on ipv6)
+# Authentication rate limit per IP for failed login attempts on unique non-existing accounts (password spraying counter-measure, per /24 on ipv4 and /48 on ipv6)
```

Comment:
We could also mention `AUTH_RATELIMIT_IP_V4_MASK` and `AUTH_RATELIMIT_IP_V6_MASK`.
What about this?

```suggestion
# Authentication rate limit per network for failed login attempts on unique non-existing accounts (password spraying counter-measure)
# Network size defaults to /24 on IPv4 and to /48 on IPv6 (see also AUTH_RATELIMIT_IP_V4_MASK and AUTH_RATELIMIT_IP_V6_MASK)
```